### PR TITLE
Plat EFR32: Update the device name supported by JLinkGDBServer.

### PR DESCRIPTION
**Pull request by xiaom-GitHub**

### CODE REVIEW

File: examples/platforms/efr32/README.md [hunk 1]
```diff
@@ -74,6 +74,9 @@ $ (gdb) monitor reset
 $ (gdb) c
 ```
 
+Note: "EFR32MG12PxxxF1024" device name has been supported by the recent JLinkGDBServer
+release, so far the latest version is V6.14d.
```
Comment (jwhui) [inline examples/platforms/efr32/README.md]: nit: suggest rewording to:

```
Note: Support for the "EFR32MG12PxxxF1024" device was added to JLinkGDBServer V6.14d.
```

Comment (xiaom-GitHub) [inline examples/platforms/efr32/README.md]: Updated, thanks.
